.github/workflows/ci.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v1
         with:
-          go-version: 1.13.4
+          go-version: 1.15
```

Comment:
Please provide this patch to cmd-template

Reply:
We already have this go version in cmd-template ci jobs